Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/interpreter/InterpreterPanel.java

```diff
@@ -513,27 +513,34 @@ private void setInputTextPaneText(String text) {
 	}
 
 	private void repositionScrollpane() {
-		// NOTE:  CRAZY CODE!  subtract one to position short of final newline
-		outputTextPane.setCaretPosition(Math.max(0, outputTextPane.getDocument().getLength() - 1));
```

Comment:
From what I understand, this is needed for the Python (really Jython) interpreter plugin. I'm not sure it's a matter of preference or a real bug, but we might revert this on our end. Have you tried using the Python plugin with this change?

Reply:
I just tried it myself with the Python window. It's definitely a matter of preference. There's an unneeded blank line between the final line of output and the prompt, which is the same behavior I see in your screenshots of the GDB window. Technically, the final line feed is there, so displaying it is probably the Right Thing, but it does waste a line. IINM, leaving our hack in has its own problems, e.g., if the final line is rather lengthy, it'll scroll the pane to the right.

Reply:
Yes, I noticed the same, but the extra spaces from the previous hack mess up the display (and can especially mess up ANSI sequences), so it is probably still better to leave the hack out - maybe there's a better way to avoid the extra blank line?